src/T2DMap.cpp

```diff
@@ -3613,16 +3662,21 @@ void T2DMap::slot_changeColor()
     hboxLayout->addWidget(pB_newColor);
 
     auto pB_ok = new QPushButton(pButtonBar);
-    pB_ok->setText(tr("OK"));
+    pB_ok->setText(tr("Ok", "dialog (room(s) change color) button"));
```

Comment:
```suggestion
    pB_ok->setText(tr("OK", "dialog (room(s) change color) button"));
```

Reply:
Interesting, https://english.stackexchange.com/q/25617  - perhaps I should just use `Accept` to avoid any capitalisation conundrums... (I do not what to have to put `O.K.`) anyone else have input on this?

Reply:
OK is enough, we're using American English.

Reply:
Okay - done in two places.